### PR TITLE
Toast messages

### PR DESCRIPTION
**Pull request by DailisLangovskis**
Added toast messages to inform user about any errors occurred while working with Compostions or Addlayers.
***NOTE** - In need of few CS translations!

### CODE REVIEW

File: projects/hslayers/src/components/data/catalogue/layman/layman.service.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ export class HsLaymanBrowserService {
   constructor(
     private http: HttpClient,
     private log: HsLogService,
-    public hsUtilsService: HsUtilsService
+    public hsUtilsService: HsUtilsService,
+    public HsToastService: HsToastService
```
Comment (raitisbe) [inline projects/hslayers/src/components/data/catalogue/layman/layman.service.ts:19]: Maybe hsToastService so its the same style at least in this file

File: projects/hslayers/src/components/data/catalogue/micka/micka.service.ts [hunk 2]
```diff
@@ -21,7 +21,8 @@ export class HsMickaBrowserService {
     private http: HttpClient,
     private log: HsLogService,
     public hsMapService: HsMapService,
-    public hsUtilsService: HsUtilsService
+    public hsUtilsService: HsUtilsService,
+    public HsToastService: HsToastService
```
Comment (raitisbe) [inline projects/hslayers/src/components/data/catalogue/micka/micka.service.ts:25]: Same here

File: projects/hslayers/src/components/layout/toast/toast.service.ts [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
+  createToastPopupMessage(
+    header: string,
+    text: string,
+    backgroundType: string
+  ): void {
+    this.show(this.HsLanguageService.getTranslation(text), {
+      header: this.HsLanguageService.getTranslation(header),
+      delay: 3000,
+      autohide: true,
+      classname: `bg-${backgroundType} text-light`,
```
Comment (raitisbe) [inline projects/hslayers/src/components/layout/toast/toast.service.ts:38]: maybe make the whole class list configurable instead? text-light will not go well together with bg-light

**Review by raitisbe — APPROVED** [review]

**Review by jmacura — COMMENTED** [review]
The toasts are disappearing far to quickly so I can read them whole. Especially when more of them appear at once (like when failing data catalogue requests). Maybe consider prolonging the time to show + adding a small close button to the top-right corner if the user is really bothered with it?

**Comment by DailisLangovskis** [comment]
> The toasts are disappearing far to quickly so I can read them whole. Especially when more of them appear at once (like when failing data catalogue requests). Maybe consider prolonging the time to show + adding a small close button to the top-right corner if the user is really bothered with it?

Ok, no problem! Will add additional time to the delay.
P.S. there already exists a close button for the toast message.

**Pull request merged**